src/features/rewards/walletPanel/index.tsx

```diff
@@ -90,25 +97,21 @@ export default class WalletPanel extends React.PureComponent<Props, {}> {
 
     return (
       <StyledSelectWrapper>
-        <Select
-          floating={true}
-          showAllContents={true}
+        <StyledSelect
           value={monthlyAmount}
-          onChange={this.props.onAmountChange}
+          onChange={this.onChange}
```

Comment:
do we need this change? Can't we just handle it as prop passed in and then in core we do `event.target.value`? Don't see real value in local `onChange` function that just calls props function

Reply:
removed